File: Modul-6/Soal-2/PRAK602-2410817210027-HafizPerdana.c
#include <stdio.h>

int main() {
    int jumlah_ruangan;
    scanf("%d", &jumlah_ruangan);

    int zetsu_putih[jumlah_ruangan];
    for (int i = 0; i < jumlah_ruangan; i++) {
        scanf("%d", &zetsu_putih[i]);
    }

    for (int i = 0; i < jumlah_ruangan; i++) {
        zetsu_putih[i] *= (i + 1);
    }

    printf("\n");
    for (int i = 0; i < jumlah_ruangan; i++) {
        printf("%d ", zetsu_putih[i]);
    }

    return 0;
}

File: Modul-6/Soal-2/PRAK602-2410817210027-HafizPerdana.py
jumlah_ruangan = int(input())

zetsu_putih = list(map(int, input().split()))

for i in range(jumlah_ruangan):
    zetsu_putih[i] *= (i + 1)

print()
for i in range(jumlah_ruangan):
    print(zetsu_putih[i], end=' ')
print()